src/Microsoft.Health.Fhir.Api/Registration/FhirServerApplicationBuilderExtensions.cs

```diff
@@ -43,11 +43,7 @@ public static class FhirServerApplicationBuilderExtensions
             EnsureArg.IsNotNull(app, nameof(app));
 
             var config = app.ApplicationServices.GetRequiredService<IOptions<FhirServerConfiguration>>();
-            var pathBase = config.Value.PathBase;
-
-            // Endpoint routing does not add trailing backslash if path is emtpy(e.g. for bundles)
-            pathBase = EnsureBackslashExists(pathBase);
-
+            var pathBase = config.Value.PathBase?.TrimEnd('/');
```

Comment:
This part looks strange. Previously we had to ensure the backslash was added. 
Now, with the new changes, we are removing the backslash if it exists.